.circleci/config.yml

```diff
@@ -48,14 +41,15 @@ commands:
             sudo apt-get update
             sudo apt-get install -y cmake curl clang llvm
             rustup component add clippy rustfmt
+            rustup toolchain install nightly
```

Comment:
Do we want to pin nightly to a date given it could break us in CI?   The overhead to maintain will be higher, but CI will be more stable, we'll have to remember to attempt to update the nightly version regularly.

Reply:
also clever use of $(CARGO)

Reply:
Good idea.